src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -1230,6 +1239,11 @@ private boolean isTVPType(Parameter[] params) throws SQLServerException {
         return false;
     }
 
+    private boolean isExecCommand(String sql) {
+        String command = sql.split(" ")[0];
```

Comment:
Not sure if this will be comprehensive. What about tabs instead of spaces? Leading spaces? Carriage returns?

I would also consider turning this around the create a check to ensure the statement adheres to the format in the JDBC spec or at least one we know will work in an RPC call.

Reply:
> What about tabs instead of spaces? Leading spaces? Carriage returns?

I was thinking of failing these cases. So, the query string must start as "EXEC ...".

> I would also consider turning this around the create a check to ensure the statement adheres to the format in the JDBC spec or at least one we know will work in an RPC call.

I'll add that change in.

Reply:
Right nvm, since spaces, tabs and carriage returns worked in the past, I'll correct it so that anything that worked in the past will work the same now.